roles/redis/defaults/main.yml

```diff
@@ -1,2 +1,3 @@
 # SPDX-License-Identifier: MIT
 ---
+redis_save_to_disk: false
```

Comment:
If the current behavior is that redis automatically saves to disk, then I think this has to be `true` by default, or it will give unexpected behavior when someone upgrades.

Reply:
Ack